### PR TITLE
Feature/image upload

### PR DESCRIPTION
**Pull request by stekug**

### CODE REVIEW

File: components/Button.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const StyledButton = styled.button<StyledButtonProps>`
   border-radius: var(--radius-m);
   height: var(--spacing-8);
   padding: var(--spacing-2);
-  padding-right: var(--spacing-3);
+  /* padding-right: var(--spacing-3); */
```
Comment (NikRoe) [inline components/Button.tsx]: If you don't need something, better remove it instead of wrapping it in a comment

```suggestion
```

**Review by NikRoe — APPROVED** [review]
Hi there,

nicely done 🙌 

The feature does what is described in the US, therefore, it gets the approval.

However, you might want to think about what happens when a user wants to edit an existing entry and change the uploaded image. Currently, this is not possible and the behaviour is not ideal:
When I edit an entry, I can select an image, but when I save the updated entry, my newly selected image is not uploaded to cloudinary.

This is something I highly encourage you to implement in your application.

Since it is not part of the current US, you can make a separate US to implement this.

Cheers ✌️

File: components/Button.tsx (hunk 1 above)
Comment (stekug) [reply components/Button.tsx]: Thanks we will address this!

File: components/RecipeForm/RecipeForm.tsx [hunk 2]
```diff
@@ -180,29 +199,63 @@ export default function RecipeForm({
...
+        ...recipeData,
+      };
+      onEditRecipe?.(updatedRecipe);
+    }
 
-    onEditRecipe?.(recipeData);
-
-    event.currentTarget.reset();
-  };
+    // event.currentTarget.reset();
```
Comment (NikRoe) [inline components/RecipeForm/RecipeForm.tsx]:
```suggestion
```

**Review by NikRoe — APPROVED** [review]
LGTM 🙌

File: components/RecipeForm/RecipeForm.tsx (hunk 2 above)
Comment (stekug) [reply components/RecipeForm/RecipeForm.tsx]: Thanks :D

**Pull request merged**